cordapp/src/test/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/CordentityE2E.kt

```diff
@@ -63,30 +71,24 @@ class CordentityE2E : CordaTestBase() {
         credentialIssuer: StartedNode<MockNode>,
         credentialProposal: String,
         credentialDefId: CredentialDefinitionId
-    ): String {
-
-        val identifier = UUID.randomUUID().toString()
-
+    ) {
```

Comment:
How am I supposed to reference the issued credential if I don't have its ID? Could you document it in a doc-comment for this method?